make/deploy-mysql

```diff
@@ -0,0 +1,126 @@
+#!/bin/bash
+set -o nounset -o errexit
+
+# This script is derived from
+#	cf-ci:qa-pipelines/tasks/usb-deploy.sh
+
+# # ## ### ##### ######## ############# #####################
+## configuration
+
+NAMESPACE="${NAMESPACE:-cf}"
+UAA_NAMESPACE="${UAA_NAMESPACE:-uaa}"
+
+USB_PLUGIN_LOCATION="https://github.com/SUSE/cf-usb-plugin/releases/download/1.0.0/cf-usb-plugin-1.0.0.0.g47b49cd-linux-amd64"
+
+SUSE_REPO="https://kubernetes-charts.suse.com"
+MYSQL_CHART="suse/cf-usb-sidecar-mysql"
+
+# For vagrant the external IP is a fixed value.
+# dig cf-dev.io (We need IP for the ASG later)
+DB_EXTERNAL_IP=192.168.77.77
+
+NFS_SCLASS="nfspersist"
+
+PASS="$(kubectl -n scf get secrets secrets -o jsonpath='{.data.cluster-admin-password}' | base64 -d)"
```

Comment:
Should be `-n ${NAMESPACE}`

Reply:
Same thing in other make scripts.

Reply:
Oops. Apologies. Will fix.

Reply:
Done with commit ddea4a434c89d631358bcd2cc7422d600cc3a62b